tests/api/test_agents_mef_rest.py

```diff
@@ -153,6 +158,26 @@ def test_agents_mef_get_updated(client, agent_mef_record, agent_idref_record,
         {}
     )
     assert res.status_code == 200
+    assert data == [{
+        'pid': agent_mef_record.pid,
+        '_created': utf_isoformat(agent_mef_record.created),
+        '_updated': utf_isoformat(agent_mef_record.updated)
+    }, {
+        'pid': agent_mef_gnd_redirect_record.pid,
+        '_created': utf_isoformat(agent_mef_gnd_redirect_record.created),
+        '_updated': utf_isoformat(agent_mef_gnd_redirect_record.updated)
+    }, {
+        'pid': agent_mef_idref_redirect_record.pid,
+        '_created': utf_isoformat(agent_mef_idref_redirect_record.created),
+        '_updated': utf_isoformat(agent_mef_idref_redirect_record.updated)
+    }]
+
+    res, data = postdata(
+        client,
+        'api_blueprint.agent_mef_get_updated',
+        {'resolve': 1}
```

Comment:
It seems that you do not test that the response contains the resolved values.

Reply:
Added